common/src/main/java/org/apache/celeborn/common/network/util/NettyUtils.java

```diff
@@ -56,13 +59,25 @@ public static ThreadFactory createThreadFactory(String threadPoolPrefix) {
     return new DefaultThreadFactory(threadPoolPrefix, true);
   }
 
-  /** Creates a Netty EventLoopGroup based on the IOMode. */
   public static EventLoopGroup createEventLoop(IOMode mode, int numThreads, String threadPrefix) {
+    return createEventLoop(mode, numThreads, false, threadPrefix);
+  }
+
+  /** Creates a Netty EventLoopGroup based on the IOMode. */
+  public static EventLoopGroup createEventLoop(
+      IOMode mode, int numThreads, boolean conflictAvoidChooserEnable, String threadPrefix) {
     ThreadFactory threadFactory = createThreadFactory(threadPrefix);
 
     switch (mode) {
       case NIO:
-        return new NioEventLoopGroup(numThreads, threadFactory);
+        return conflictAvoidChooserEnable
```

Comment:
so the issue only happens on NIO mode? seems the PR description hasn't reflected it.

Reply:
nvm, I see https://github.com/apache/celeborn/pull/3220 applied it to EPOLL branch too.